Add the Create List into the documentation (~2447) (#125)
simplic/documentation: support/Misc/Forms/forms.md

When the user clicks "Speichern und schließen" in the main window, the form is saved.  

## Create a List
When the user creates a new Field then the user must set the "Datentyp" to List.  

![AddListTreeView](~/images/Forms/addListTreeView.png)  

> With a right click on a "List Field" the user can create a Child Field, when the user clicks "Neues Feld".  

Reviewer: ```suggestion
To create a child field the user clicks on "Neues Feld" inside the context menu, which is opened by right clicking on a "List Field".
```

> If the user will update a Field, then clicks "Feld bearbeiten".  

Reviewer: ```suggestion
To change a child field the user clicks on "Feld bearbeiten" inside the context menu.
```

![AddListTreeViewChild](~/images/Forms/addListTreeViewChild.png)  

> When a Field is unnecessary then the user clicks "Null-Werte zulassen", so the Field can be empty.  

Reviewer: ```suggestion
When a Field is not required the user enables "Null-Werte zulassen", so the field can be empty.  
```

![AddListTreeViewChildNullable](~/images/Forms/addListTreeViewChildNullable.png)  

The user selects the item "GridViewTemplate" from the tool box "Simplic.Forms.UI.dll" and drags it to the designer surface.  

![DesignTabList](~/images/Forms/designTabList.png)  

In the attribute list the attribut "ItemSource" will be expanded.  
Three attributes are very important.  
**Mode** it must be TwoWay  
**Path** the name of the data from the "Datenstruktur"  
**UpdateSourceTrigger** the default value is "PropertyChanged" or for text it is "LostFocus"  
*PropertyChanged:* Updates the binding source immediately whenever the binding target property changes.  
*LostFocus:* Updates the binding source whenever the binding target element loses focus.  

![ImportantListBinding](~/images/Forms/importantBindingList1.png)  

> The user clicks in the Property "Columns" the button "..." and can add Columns to the Grid.  

Reviewer: ```suggestion
Clicking on the "..." button for the property "Columns" the user can add columns to the grid.  
```

![ListColumnCreateClick](~/images/Forms/listColumnCreateClick.png)  

The user selects "Telerik.Windows.Controls.GridViewDataColumn" from the combo box and then clicks the button "Add".  

![ListColumnAdd](~/images/Forms/listColumnAdd.png)  

For each Column from the Grid the user must fill these Attributes.  
*Name* The name of the GridViewDataColumn.  
*Header* The name of the header in the GridView.  

![ListColumnAddDesign](~/images/Forms/listColumnAddDesign.png)  

The last step is binding to the DataMemberBinding.  
Three attributes are very important.  
**Mode** it must be TwoWay  
**Path** the name of the column which is the same from the "Datenstruktur".   
**UpdateSourceTrigger** the default value is "PropertyChanged" or for text it is "LostFocus"  
*PropertyChanged:* Updates the binding source immediately whenever the binding target property changes.  
*LostFocus:* Updates the binding source whenever the binding target element loses focus.  

![ListColumnBinding](~/images/Forms/listColumnBinding.png)  

## Create Database Table

> When the user clicks "Tabelle hinzufügen/bearbeiten" a new Database table will be create.  

Reviewer: ```suggestion
When the user clicks "Tabelle hinzufügen/bearbeiten" a new Database table will be created.  
```
